### ISSUE TITLE
Is anybody working on a web frontend or other GUI?

### ISSUE THREAD

**Issue by mponweiser** [jrnewell/spotify-ripper#266]

**Comment by jonaaa20** [comment]
Is that hard to use it? i think a GUI is not needed (for now)...

**Comment by mponweiser** [comment]
I don't think it would be an essential feature, but surely I would prefer something more convenient than putty/screen/manual pasting. I was just checking in before even considering working on a flask app, in order to avoid reinventing the wheel.

Anyway, no more information or action needed from my side at the moment because I think such a project would require more effort than I would be willing to invest right now.

**Comment by AlexOwen** [comment]
I think changing the --log mode (or adding a verbose switch) to include precent complete would make it much easier to parse the output and to make a GUI. The current version looks great but isn't pleasant to parse, and the --log switch is quite quiet (which is good for a log).

**Comment by orcomma** [comment]
Just my opinion: web front end is good for individual URIs but unless it has an upload feature of some sort it would be counter intuitive for ripping files of multiple URIs.  The CLI interface is perfect IMHO
